### PR TITLE
Ms 2002.4 guesh

### PR DESCRIPTION
**Pull request by GueshSol**

### CODE REVIEW

File: new/CIA20024.xml [hunk 1]
```diff
@@ -0,0 +1,927 @@
...
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                
+                                
+                                <msItem xml:id="ms_i2.2.32.3">
+                                    <locus target="#139r"/>
+                                    <title type="complete" ref="LIT5673AkkonuColl#AkkMin32"/>
+                                    <textLang mainLang="gez"/>
+                                </msItem>
+                                <msItem xml:id="ms_i2.2.33">
```
Comment (thea-m) [inline new/CIA20024.xml:883]: This item is included in ms_i2.2.32, but numbered ms_i2.2.33: It should either be numbered ms_i2.2.32.4 or be moved outside of ms_i2.2.32. Both solutions would be fine, you have to decide which is conceptually better.

**Review by thea-m — CHANGES_REQUESTED** [review]
I think that everything is fine - I did not double check the IDs now, but all seems good. Only one remark in the last item. Thank you!

**Review by thea-m — APPROVED** [review]

**Comment by thea-m** [comment]
@GueshSol note that I now moved this manuscript from the folder "new" to the folder "ChicagoIA"

**Comment by GueshSol** [comment]
Thank you very much.

**Pull request merged**